docs/carts/cart_interactions/add_packages.md

```diff
@@ -16,6 +16,7 @@ This will add all of the package's included items to the cart.
 | Field name                         | Description                                            |
 | ---------------------------------- | ------------------------------------------------------ |
 | `package[id]`                      | The ID of the package to add to the cart.              |
+| `package[quantity]`                | The quantity of the package to add to the cart.        |
```

Comment:
```suggestion
| `package[quantity]`                | The number of the packages to add to the cart at once.        |
```

Can we add a note above stating that the quantity field should only be used for packages that don't have booking journeys